docs/user_workloads/accessing_virtual_machines.md

```diff
@@ -327,29 +330,30 @@ and traffic consider using a dedicated Kubernetes `Service` instead.
 2. SSH into virtual machine
 
 ```shell
-# Add --local-ssh to transparently use local OpenSSH client
-virtctl ssh -i id_rsa fedora@testvm
+virtctl ssh -i id_rsa fedora@vm/testvm/mynamespace
 ```
 
 or
 
 ```shell
-ssh -o 'ProxyCommand=virtctl port-forward --stdio=true vmi/testvm.mynamespace 22' -i id_rsa fedora@vmi/testvm.mynamespace
+ssh -o 'ProxyCommand=virtctl port-forward --stdio=true vm/testvm/mynamespace 22' -i id_rsa fedora@vm/testvm/mynamespace
 ```
 
-4. SCP file to the virtual machine
+3. SCP file to the virtual machine
 
 ```shell
-# Add --local-ssh to transparently use local OpenSSH client
-virtctl scp -i id_rsa testfile fedora@testvm:/tmp
+virtctl scp -i id_rsa testfile fedora@vm/testvm/mynamespace:/tmp
 ```
 
 or
 
 ```shell
-scp -o 'ProxyCommand=virtctl port-forward --stdio=true vmi/testvm.mynamespace 22' -i id_rsa testfile fedora@testvm.mynamespace:/tmp
+scp -o 'ProxyCommand=virtctl port-forward --stdio=true vmi/testvm/mynamespace 22' -i id_rsa testfile fedora@vm.testvm.mynamespace:/tmp
 ```
 
+Note that local `scp` does not support slashes in hostnames,
+therefore the example uses dots to separate the type, name and namespace.
```

Comment:
Dumb question: does the reader also need to use `.` to separate objects in their scp command?
If so, might be worth being more direct, i.e.:
```suggestion
!!! Note 
    Local `scp` does not support slashes in hostnames. You need to use `.` to separate the type, name, and namespace.
```

Reply:
Technically they don't need to. They can chose whatever remote hostname they would like, as long as the `ProxyCommand` is set up correctly. However, it probably makes sense to stick to what virtctl does.